### ISSUE TITLE
How to use brainstat.stats.SLM with parcellated data? P-values are missing

### ISSUE THREAD

**Issue by JohannesWiesner** [MICA-MNI/BrainStat#364]
* **What are you trying to do?**

TLDR; Can BrainStat really handle parcellated surface data?

Hi! If I read the brainstat-paper correctly, it should also be possible to set up a linear model with already-parcellated data? Specifically I would like to use the 68 regions from the Desikan-Killiany Atlas (as provided from freesurfer in the `lh/rh.aparc.stats` files). I tried to follow the [brainstat tutorial](https://brainstat.readthedocs.io/en/master/python/tutorials/tutorial_1.html#python-tutorial1). The only difference between the tutorial and my dataset is that I don't have vertex-data but parcel-wise data, which means that my `thickness` object has only 68 entries. 

* **If you've tried running this with BrainStat code already, how did you do so and what went wrong?**

My code currently is more of less just a copy of the tutorial code. The only difference here is that `thickness` only has 68 entries (because it represents average thickness across the Desikan-Killiany parcels). I set this as a model:

```python
slm_age = SLM(
    model=model,
    contrast=contrast_age,
    surf=None,
    mask=None,
    correction='fdr'
)
```

since `surf='fsaverage5'` does not work in my case and setting `correction='rft'` would also not make sense in the case of parcellated data? The .fit() method does not throw an error but `slm_age.P` is `None` in my case.

* **Please tell us about your computing environment:**
  
* **Other information**

**Comment by zihuaihuai** [comment]
Briefly, yes, BrainStat can handle parcellated surface data.

In your case, since you're not using fsaverage5, setting surf to this value is incorrect. You can refer to the documentation to prepare your surface appropriately: [BrainStat Documentation](https://brainstat.readthedocs.io/en/master/python/_autosummary/brainstat.stats.SLM.SLM.html#brainstat.stats.SLM.SLM).

As for the correction parameter, it's used for multiple comparison corrections. In the simplest case, you could try running the model without applying any correction first.
